### PR TITLE
Add firebase performance and analytics

### PR DESCRIPTION
**Pull request by davwheat**

### CODE REVIEW

File: packages/gatsby-plugin-firebase/src/index.js [hunk 1]
```diff
@@ -32,6 +35,7 @@ function Index({ features, children }) {
           storageBucket: process.env.GATSBY_FIREBASE_STORAGE_BUCKET,
           messagingSenderId: process.env.GATSBY_FIREBASE_MESSAGING_SENDER_ID,
           appId: process.env.GATSBY_FIREBASE_APP_ID,
+          measurementId: process.env.GATSBY_FIREBASE_MEASUREMENT_ID ? process.env.GATSBY_FIREBASE_MEASUREMENT_ID : null,
```
Comment (alexluong) [inline packages/gatsby-plugin-firebase/src/index.js]: You can just use `process.env.GATSBY_FIREBASE_MEASUREMENT_ID`. I think `undefined` value is fine too when initializing the Firebase app.

**Comment by alexluong** [comment]
Thanks for the PR. Everything looks good. I added a small comment. It should be ready to merge and publish a new version whenever you want. What do you think?

**Comment by davwheat** [comment]
Sounds great! Fixed the issue you flagged.

**Comment by alexluong** [comment]
@davwheat Awesome. Merged and published!

**Pull request merged**